### PR TITLE
replace new HtmlView(String) with HtmlView.of()/unsafe()/err()

### PR DESCRIPTION
**Pull request by labkey-matthewb**
#### Rationale
Introduce HtmlView.of(), HtmlView.unsafe(), and HtmlView.err() to help provide a migration path of the deprecated constructor HtmlView(String)

#### Related Pull Requests
* 

#### Changes
*

### CODE REVIEW

File: api/src/org/labkey/api/query/QueryWebPart.java [hunk 1]
```diff
@@ -261,11 +261,11 @@ protected void addViews()
         {
             if (_schemaName == null)
             {
-                view = new HtmlView("Schema name is not set.");
+                view = HtmlView.unsafe("Schema name is not set.");
```
Comment (labkey-adam) [inline api/src/org/labkey/api/query/QueryWebPart.java]: Why not `HtmlView.of()`?

File: api/src/org/labkey/api/reports/report/r/view/FileOutput.java [hunk 2]
```diff
@@ -61,7 +61,7 @@ public HttpView getView(ViewContext context)
         if (getReport() instanceof AttachmentParent)
             return new FileoutReportView(this, getReport());
         else
-            return new HtmlView(DownloadParamReplacement.UNABlE_TO_RENDER);
+            return HtmlView.of(DownloadParamReplacement.UNABlE_TO_RENDER);
```
Comment (labkey-adam) [inline api/src/org/labkey/api/reports/report/r/view/FileOutput.java]: Not a new thing, but it there really a lowercase "L" in the middle of this var?

**Review by labkey-adam — CHANGES_REQUESTED** [review]
`QueryWebPart` should use `HtmlView.of()` for consistency, me thinks

File: api/src/org/labkey/api/query/QueryWebPart.java (hunk 1 above)
Comment (labkey-matthewb) [inline api/src/org/labkey/api/query/QueryWebPart.java]:
```suggestion
                view = HtmlView.of("Schema name is not set.");

```

File: api/src/org/labkey/api/reports/report/r/view/FileOutput.java (hunk 2 above)
Comment (labkey-matthewb) [reply api/src/org/labkey/api/reports/report/r/view/FileOutput.java]: Assign a bug to Dax

**Review by labkey-adam — APPROVED** [review]
Lovely

**Pull request merged**